packages/gestalt/src/AvatarGroup.js

```diff
@@ -47,7 +90,7 @@ const AvatarGroupWithForwardRef: React$AbstractComponent<Props, UnionRefs> = for
     onClick,
     role,
     size = 'fit',
-  },
+  }: Props,
```

Comment:
@AlbertCarreras this was the "trick" to get the auto generated docs to work. Manually specifying the `Props` on the component.

Reply:
I wonder if we should keep a list somewhere of these unintuitive "gotchas" we're figuring out about getting auto-generated props to work?